data/scripts.js

```diff
@@ -157,154 +165,150 @@ exports.BattleScripts = {
 			// Adjust again
 			target = this.resolveTarget(pokemon, move);
 		}
-		if (!move) return false;
+		if (move && !pokemon.fainted) {
+			let attrs = '';
```

Comment:
Reindenting literally everything is kind of bad.

Why not just do
```
if (!move) return false;
if (pokemon.fainted) {
  // do whatever other handling necessary
  return false;
}

Reply:
That actually save any indents, though, does it?

Reply:
Yes? Early returns are the usual way to unindent massive blocks. Compare

```
if (test) {
  // massive block
}
```

vs

```
if (!test) return;
// massive block
```

Reply:
yeah in that case it does, but here it's

    if (move && !pokemon.fainted) {
      // do whatever other handling necessary
      return false;
    }

vs. 

    if (!move) return false;
    if (pokemon.fainted) {
      // do whatever other handling necessary
      return false;
    }

Reply:
That's not what I'm suggesting. Currently it's:

```
if (move && !pokemon.fainted) {
  // huge block
}
// cleanup
```

I'm suggesting

```
if (!move) return;
if (!pokemon.fainted) {
  // cleanup
  return;
}
// huge block
// cleanup
```

Reply:
Ah, OK, but shouldn't that be

    if (!move || pokemon.fainted) {
      // cleanup
      return;
    }
    // huge block
    // cleanup

Reply:
Judging from the diff, the `!move` case doesn't actually need cleanup, but I might be wrong in which case yours works too.

Reply:
The // cleanup includes both the check for whether a submove happened and the actual storing of the value, so it needs to happen at every return.

Reply:
But I guess we don't need to do the check if the move doesn't exist or the Pokemon's fainted